### ISSUE TITLE
Still unable to delete web apps if the .app was manually deleted beforehand

### ISSUE THREAD

**Issue by yllekz** [filips123/PWAsForFirefox#492]
### Description

Continuation of #427, as it was marked as fixed but the same bug prevails.

### Steps to Reproduce

    1. Create a PWA
    1. Delete the .app
    1. Try to delete PWA from Firefox

### Additional Information
n/a

### Affected Websites

    Facebook
    Instagram
    YouTube
    YouTube TV
    Self-hosted websites

### Environment

* Operating system: macOS Sonoma 14.4.1
* System architecture: M2
* Desktop environment: 
* Installation method: Homebrew
* PWAsForFirefox extension version: 2.11.1
* PWAsForFirefox native version: 2.11.1
* PWAsForFirefox runtime version: 125.0.2
* Firefox version: 125.0.1

**Comment by filips123** [comment]
I can't reproduce this on macOS 10.15.7. Maybe it's specific to newer macOS versions?

And I also don't know why this happens. There is now [a check](https://github.com/filips123/PWAsForFirefox/blob/57cbf787db4dab43d4254f88c2627239b62a6520/native/src/integrations/implementation/macos.rs#L553-L555) to skip removing the bundle if it doesn't exist.

**Comment by yllekz** [comment]
Hello - just checking in and confirming this is still an active bug that I can reproduce. It's rather frustrating that I can't delete a PWA.

**Comment by filips123** [comment]
So, still, if you remove the `.app` bundle manually, then then try to uninstall web app from the extension, it fails with "No such file or directory"?

Please create a `DEBUG` file and then collect the native logs as described [here](https://pwasforfirefox.filips.si/help/troubleshooting/#obtaining-native-logs).

**Comment by yllekz** [comment]
Yes, and I can confirm this is still happening today. I can't delete any of the PWAs I created. I'll attempt to pull some logs.
